examples/job/get-file-from-job.py

```diff
@@ -0,0 +1,201 @@
+# Espresso to your account. Finally, a job is created using the Quantum Espresso workflow for the silicon unit cell,
+# and the job is submitted to the cluster. For more information, please refer to our [
+# run-simulation-and-extract-properties](./run-simulations-and-extract-properties.ipynb) notebook, located in this directory.
+
+# In[]:
+
+
+# Get some account information
+project_endpoints = ProjectEndpoints(*ENDPOINT_ARGS)
+project_metadata = project_endpoints.list({"isDefault": True, "owner._id": ACCOUNT_ID})[0]
+project_id = project_metadata['_id']
+owner_id = project_metadata['owner']['_id']
+
+# Get a workflow for the job from the bank, and copy it to our account
+bank_workflow_endpoints = BankWorkflowEndpoints(*ENDPOINT_ARGS)
+BANK_WORKFLOW_ID = "84DAjE9YyTFndx6z3"
+workflow_id = bank_workflow_endpoints.copy(BANK_WORKFLOW_ID, owner_id)["_id"]
+
+# Get materials for the job
+material_endpoints = MaterialEndpoints(*ENDPOINT_ARGS)
+material_project_id = ["mp-149"]  # The importer expects a list
+materials = material_endpoints.import_from_materialsproject(MATERIALS_PROJECT_API_KEY, material_project_id, owner_id)
+
+# Create the job
+job_endpoints = JobEndpoints(*ENDPOINT_ARGS)
+job = job_endpoints.create_by_ids(materials=materials,
+                                  workflow_id=workflow_id,
+                                  project_id=project_id,
+                                  owner_id=owner_id,
+                                  prefix="Test_Job_Output")[0]
+
+# Submit the job
+job_endpoints.submit(job['_id'])
+wait_for_jobs_to_finish(job_endpoints, [job['_id']])
+
+# Monitor the jobs and print the status until they are all finished.
+
+# ## Viewing job files
+# ### Retreive a list of job files
+# 
+# Here, we'll get a list of all files that belong to the job.
+
+# In[]:
+
+
+files = job_endpoints.list_files(job['_id'])
+paths = [file['key'] for file in files]
+for path in paths:
+    if 'outdir' not in path:
+        print(path)
+
+# ### Get metadata for the Output File
+# The .out file is where Quantum Espresso shows its work and prints its results, so you most likely will want to view this
+# files. Let's print out some of its metadata.
+# 
+# You'll find that we get a lot of data describing the file and its providence. Brief explanations of each entry are:
+# - Key - Path to the file on the cluster
+# - size - Size of the file, in bytes.
+# - Bucket - The name of the cluster which ran the job.
+# - Region - Which server region was used to run the job.
+# - Provider - The cluster provider for the compute resources (in our case, we used AWS).
+# - lastModified - Unix timestamp representing when the file was last modified.
+# - name - The filename.
+# - signedUrl - This is a link which can be used to download the file for a short amount of time.
+
+# In[]:
+
+
+for file in files:
+    if file['name'] == 'pw_scf.out':
+        outfile_metadata = file
+JSON(outfile_metadata)
```

Comment:
It's a nit, but I would generally recommend against using shortened names in your code - the overhead of typing the full name once is lower than having to allocate brainpower each time to "unshorten" them.

outfile -> output_file
infile_bytes -> input_file_bytes